code/datums/diseases/advance/advance.dm

```diff
@@ -426,33 +425,76 @@
 
 //This proc is used when creating diseases, to call OnAdd for each symptom to make sure the symptoms work as they should
 /datum/disease/advance/proc/Finalize()
+	RemoveWildSymptom()
 	for(var/datum/symptom/S in symptoms)
 		S.OnAdd(src)
 
 
 // Add a symptom, if it is over the limit we take a random symptom away and add the new one.
-/datum/disease/advance/proc/AddSymptom(datum/symptom/S)
+/datum/disease/advance/proc/AddSymptom(datum/symptom/S, remove_wild=TRUE)
+	if(remove_wild)
+		RemoveWildSymptom()
 
 	if(HasSymptom(S))
 		return
 
-	if(symptoms.len >= VIRUS_SYMPTOM_LIMIT)
-		RemoveSymptom(pick(symptoms))
+	if(symptoms.len >= VIRUS_SYMPTOM_LIMIT+VIRUS_WILD_COUNT)
+		var/datum/symptom/target
+		for(var/count in 1 to 150)
+			target = pick(symptoms)
+			if(!istype(target, /datum/symptom/wild))
+				break
+			if(count == 150)
+				return
```

Comment:
I know this code is not beautiful, but trying a pick from a list without a specific object was not possible to me.